pkg/apis/work/v1alpha1/binding_types.go

```diff
@@ -97,6 +98,7 @@ type ResourceBindingList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:scope="Cluster"
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=crb
```

Comment:
I‘d prefer to put `shortName` and `scope` together, like 
```
// +kubebuilder:resource:scope="Cluster",shortName=crb
```

Reply:
That also explains the CI failure.

Reply:
done